### ISSUE TITLE
Trouble with Chrome/Plunker

### ISSUE THREAD

**Issue by hijustin** [gordonmzhu/questions#297]
### What is the associated Watch and Code lesson? 

https://watchandcode.com/courses/60264/lectures/936334

### Include a live demo link so that I can run your code. 

https://plnkr.co/edit/jtMh0b?p=preview

### Explain the exact steps I need to take to replicate the problem. 

I've typed out the code in the editor as shown in the video, inspected/opened console, filtered by plunkerPreviewTarget, and typed 
todoList.displayTodos(); in the console and hit Enter.

### What did you expect to happen? 

I expected the console to run the displayTodos method.

### What actually happened? 

I get the error "Uncaught ReferenceError: displayTodos is not defined
    at <anonymous>:1:1"

### Describe what you've done so far to troubleshoot the problem. 

I've triple-checked my code, resaved the Plunk, refreshed/restarted the browser.

**Comment by gordonmzhu** [comment]
Just fired up your app and ran `todoList.displayTodos()` and it worked fine. Maybe try it again?
<img width="1280" alt="Screen Shot 2019-09-25 at 4 53 50 PM" src="https://user-images.githubusercontent.com/734575/65647953-369f6080-dfb5-11e9-9946-5553b0524948.png">

**Comment by hijustin** [comment]
Hi Gordon, thanks for taking a look at this! I tried it again after you
checked it and it worked fine for the first time. I didn't change a thing.
Then I came back later and now I'm getting the same error again. I thought
it might be due to Chrome extensions so I turned them all off.. still
getting the error. I'm confused as to why it would work one day and not the
next when I haven't changed anything. I appreciate you taking the time to
look-- I just really want to understand if it's something I'm doing wrong
or what. I'm pulling my hair out over here! Here is a link to the plunk
again:  https://plnkr.co/edit/jtMh0b?p=preview

Justin
